app/ui.js

```diff
@@ -734,7 +734,7 @@ var UI = {
 
         // Save the cookie for this session
         if (typeof value !== 'undefined') {
-            WebUtil.writeSetting(name, value);
+            WebUtil.writeSetting(name, value, true);
```

Comment:
The only place that provides a `value` is the call from `initSetting()`. So I think we can take this opportunity to move this chunk to `initSetting()` and get rid of the `value` argument.